src/compas_fab/robots/planning_scene.py

```diff
@@ -248,7 +248,7 @@ def __init__(self, robot):
 
     @property
     def client(self):
-        """:class:`compas_fab.backend.RosClient` or :class:`compas_fab.backend.VrepClient` : The backend client."""
+        """:class:`compas_fab.backend.RosClient`: The backend client."""
```

Comment:
This should change to the `ClientInterface`, right?

Reply:
yep